From 599e805def46419d6e9e9ba515fa24cedfa90e24 Mon Sep 17 00:00:00 2001
From: Karsten Merker <merker@debian.org>
Date: Fri, 11 Sep 2015 16:27:47 +0200
Subject: [PATCH] Enable the Linksprite_pcDuino target in u-boot-sunxi.

---
 debian/targets | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/targets b/debian/targets
index 7208078..0fa4445 100644
--- a/debian/targets
+++ b/debian/targets
@@ -95,6 +95,9 @@ armhf	sunxi		Cubieboard2	u-boot-sunxi-with-spl.bin
 # Karsten Merker <merker@debian.org>
 armhf	sunxi		Cubietruck	u-boot-sunxi-with-spl.bin
 
+# Robert Hegner <rhegner@hsr.ch>
+armhf	sunxi		Linksprite_pcDuino	u-boot-sunxi-with-spl.bin
+
 # Patrice Go <patgsiosisr@gmail.com>
 armhf	sunxi		Linksprite_pcDuino3	u-boot-sunxi-with-spl.bin
 
-- 
2.5.1

