diff -ur u-boot-2018.05+dfsg/debian/targets u-boot-2018.05+dfsg.eMMC/debian/targets
--- u-boot-2018.05+dfsg/debian/targets	2018-05-10 21:59:27.000000000 +0200
+++ u-boot-2018.05+dfsg.eMMC/debian/targets	2018-06-17 15:43:03.539088901 +0200
@@ -112,6 +112,9 @@
 # Karsten Merker <merker@debian.org>
 armhf	sunxi		A20-OLinuXino-Lime2	u-boot-sunxi-with-spl.bin
 
+# Andreas B. Mundt <andi@debian.org>
+armhf	sunxi		A20-OLinuXino-Lime2-eMMC	u-boot-sunxi-with-spl.bin
+
 # Arne Ploese <aploese@gmx.de>
 armhf  sunxi		A20-OLinuXino_MICRO	u-boot-sunxi-with-spl.bin
 
diff -ur u-boot-2018.05+dfsg/debian/u-boot-sunxi.lintian-overrides u-boot-2018.05+dfsg.eMMC/debian/u-boot-sunxi.lintian-overrides
--- u-boot-2018.05+dfsg/debian/u-boot-sunxi.lintian-overrides	2018-05-10 21:07:03.000000000 +0200
+++ u-boot-2018.05+dfsg.eMMC/debian/u-boot-sunxi.lintian-overrides	2018-06-17 08:38:45.032905770 +0200
@@ -8,6 +8,7 @@
 u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/A20-Olimex-SOM-EVB/uboot.elf
 u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/A20-OLinuXino-Lime/uboot.elf
 u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/A20-OLinuXino-Lime2/uboot.elf
+u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/A20-OLinuXino-Lime2-eMMC/uboot.elf
 u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/A20-OLinuXino_MICRO/uboot.elf
 u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/Bananapi/uboot.elf
 u-boot-sunxi [armhf]: arch-dependent-file-not-in-arch-specific-directory usr/lib/u-boot/Bananapro/uboot.elf
@@ -32,6 +33,7 @@
 u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/A20-Olimex-SOM-EVB/uboot.elf
 u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/A20-OLinuXino-Lime/uboot.elf
 u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/A20-OLinuXino-Lime2/uboot.elf
+u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/A20-OLinuXino-Lime2-eMMC/uboot.elf
 u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/A20-OLinuXino_MICRO/uboot.elf
 u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/Bananapi/uboot.elf
 u-boot-sunxi [armhf]: statically-linked-binary usr/lib/u-boot/Bananapro/uboot.elf
